components/ecu/ecu_firmware/Core/Src/can.c

```diff
@@ -4,21 +4,31 @@
  * 
  * @date 2021/01/30
  * @author Blake Shular (blake-shular)
+
+ * @param currentVal the current flowing through the ECU
+ * @param doc_coc Discharge and Charge overcurrent status
```

Comment:
`@param` descriptions go in a comment specifically for the function, immediately above the function definition. Where you've put these is the file description, not a function description. Please take another look at my example in a previous comment. Just FYI, these function description comments are formatted according to a documentation tool called [Doxygen](https://www.doxygen.nl/manual/docblocks.html). We don't really use the tool though, we just use its formatting standard.

Also, please add more details to your descriptions. What are the units of `currentVal` supposed to be? What values are allowed for `doc_coc` and what does each mean?  The point of these descriptions is to inform someone else how to use this function correctly.